sktime/utils/validation/__init__.py

```diff
@@ -13,6 +13,12 @@ def is_int(x):
     return isinstance(x, (int, np.integer)) and not isinstance(x, bool)
 
 
+def is_float(x):
+    """Check if x is of integer type, but not boolean"""
+    # boolean are subclasses of integers in Python, so explicitly exclude them
+    return isinstance(x, (float, np.floating)) and not isinstance(x, bool)
```

Comment:
booleans are not floats 
```
    return isinstance(x, (float, np.floating))
```

Reply:
Thank you @mloning . I have made changes to it